src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -124,7 +125,29 @@ public MesosCloud(
       boolean checkpoint,
       boolean onDemandRegistration,
       String jenkinsURL) throws NumberFormatException {
-    super("MesosCloud");
+    this("MesosCloud", nativeLibraryPath, master, description, frameworkName, slavesUser, 
```

Comment:
Ok. I have seen the changes and the constructor (this(....)) was modified too, for this reason I thought that this issue had introduced this.